src/panels/lovelace/components/hui-image.ts

```diff
@@ -119,16 +129,25 @@ class HuiImage extends LitElement {
           ratio: Boolean(ratio && ratio.w > 0 && ratio.h > 0),
         })}
       >
-        <img
-          id="image"
-          src=${imageSrc}
-          @error=${this._onImageError}
-          @load=${this._onImageLoad}
-          style=${styleMap({
-            filter,
-            display: this._loadError ? "none" : "block",
-          })}
-        />
+        ${this.cameraImage && this.cameraView === "live"
+          ? html`
+              <ha-camera-stream
+                .hass="${this.hass}"
+                .stateObj="${cameraObj}"
+              ></ha-camera-stream>
+            `
+          : html`
+              <img
```

Comment:
Probably for another PR, but I feel like we should consider putting this in it's own component too.